### PR TITLE
medium: bootstrap: disable completion and history when running bootstrap

### PR DESCRIPTION
**Pull request by liangxin1300**
When running crm->cluster->init(or other bootstrap function),
  completion makes no sense, it's better to disable it

I think this is a better and clear way against #218 
1) When meet multi_input and ask in bootstrap, still have completion handler, but the handler will complete nothing
2) remove the history item after inputs in bootstrap, because these items cannot use for other crmsh's commands and levels

### CODE REVIEW

**Comment by liangxin1300** [comment]
The issue is that, when other commands instead of bootstrap calling utils.ask, completion still not necessary

should we pass the context into the utils.ask to disable it's completion?

File: crmsh/bootstrap.py [hunk 1]
```diff
@@ -105,9 +106,13 @@ def prompt_for_string(msg, match=None, default='', valid_func=None):
     if _context.yes_to_all:
         return default
     while True:
+        disable_completion()
```
Comment (krig) [inline crmsh/bootstrap.py]: Shouldn't this be `_context.disable_completion()`?

Comment (liangxin1300) [inline crmsh/bootstrap.py]: no, disable_completion() is a local **wrap** function in line 134:)

Comment (krig) [inline crmsh/bootstrap.py]: Ah, I missed it :)

**Pull request merged**